charts/vault-monitoring/values.yaml

```
@@ -50,6 +50,17 @@ prometheusRules:
   # prometheusRules.rules -- set of prometheus alerts to define
   # @default -- list of predefined alerts
   rules:
+    - alert: VaultAutopilotNodeHealthy
+      # Set to 1 if the node_id is deemed healthy by Autopilot, 0 if not
+      # node_id: Unique identifier of a raft peer, same as peer_id
+      # https://www.vaultproject.io/docs/internals/telemetry#integrated-storage-raft-autopilot
+      expr: count by (instance) (vault_autopilot_node_healthy) < 5
```

```suggestion
      expr: vault_autopilot_healthy < 1
```

As per https://www.vaultproject.io/docs/internals/telemetry#integrated-storage-raft-autopilot we can just use `vault_autopilot_healthy`

---

Fixes #611

---

Thanks, that would also resolve @pree question regarding the node count. I'll test this on a cluster to observe the behavior..

---

This works perfectly fine, I tested it. Also, changed the description to make clear that the check still has the same "amount of information" (it checks the health of all the nodes and alerts "if at least one node is considered unhealthy"), even without the explicit threshold.